homeassistant/components/teslemetry/coordinator.py

```diff
@@ -62,21 +65,42 @@ def __init__(
         )
         self.api = api
         self.data = flatten(product)
+        self.pre2021 = product["vin"][9] <= "L"
```

Comment:
Would it make sense for the "age check" to live in the library instead?
Seems a bit weird to parse the VIN in the coordinator.

Reply:
Sure, I'll add this line as a prop in the library.

Reply:
It's still here?

Reply:
Must have snuck back in during a rebase or something. Its removed.